<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于STM32的温湿度控制系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sourse.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
border-image: url(:/images/lver.gif);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_t_h_ip">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>95</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温湿度IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_t_h_IP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_t_h_Port">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_t_h_Port">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>88</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ill_ip">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>121</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光照IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ill_IP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ill_Port">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>116</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ill_Port">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>114</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fan_ip">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>147</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>风扇IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fan_IP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fan_Port">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>142</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fan_Port">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>171</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_t">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>174</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_t_alarm">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>172</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_humidity">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_h">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>203</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_h_alarm">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>201</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_illumination">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>229</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光照</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_ill">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>232</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ill_alarm">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_blue_LED">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>蓝灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yellow_LED">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>黄灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_green_LED">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>绿灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>风扇</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_system_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="digitCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_humidity_set">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>湿度阈值：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_illumination_set">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>光照阈值：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_temperature_set">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>温度阈值：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_t">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_h">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_ill">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>progressBar_ill</zorder>
   <zorder>label_temperature</zorder>
   <zorder>progressBar_t</zorder>
   <zorder>label_illumination</zorder>
   <zorder>label_humidity</zorder>
   <zorder>progressBar_h</zorder>
   <zorder>label_t_h_ip</zorder>
   <zorder>label_t_h_Port</zorder>
   <zorder>lineEdit_t_h_Port</zorder>
   <zorder>lineEdit_t_h_IP</zorder>
   <zorder>lineEdit_ill_IP</zorder>
   <zorder>lineEdit_ill_Port</zorder>
   <zorder>label_ill_Port</zorder>
   <zorder>label_ill_ip</zorder>
   <zorder>label_fan_ip</zorder>
   <zorder>lineEdit_fan_IP</zorder>
   <zorder>label_fan_Port</zorder>
   <zorder>lineEdit_fan_Port</zorder>
   <zorder>pushButton_t_alarm</zorder>
   <zorder>pushButton_h_alarm</zorder>
   <zorder>pushButton_ill_alarm</zorder>
   <zorder>pushButton_blue_LED</zorder>
   <zorder>pushButton_yellow_LED</zorder>
   <zorder>pushButton_green_LED</zorder>
   <zorder>pushButton_fan</zorder>
   <zorder>lcdNumber_system_time</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sourse.qrc"/>
 </resources>
 <connections/>
</ui>
